ci_tests/cfg-indent-check.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+import os
+import re
+import sys
+
+
+def cfg_indention_check():
+    exit_code = 0
+    for file in sys.argv[1:]:
+        status_code = 0
+        cfg_path = os.path.abspath(os.path.join(os.path.dirname(__file__),
+                                                os.path.pardir, file))
+        with open(cfg_path, "r") as f:
+            contents = f.read().splitlines()
+            for num, line in enumerate(contents, 1):
+                stripped_line = line.lstrip(" ")
+                if (len(line) - len(stripped_line)) % 4:
+                    print(f"{file}:{num} doesn't match 4x space indentation")
```

Comment:
I'd suggest using the relative filepath to point out the problematic file so that users can locate the file directly.

Reply:
Yes, now the filename is relative path, like `qemu/tests/cfg/xxxx.cfg`